### PR TITLE
Tech event site

### PR DESCRIPTION
**Pull request by kosher9**
- Created the mobile version
- Created the desktop version
- Added navigation from home to about page

Demo of the tech_event website: https://www.loom.com/share/f070f19ee92b4a638e7e6a8e9aeae0f3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                        Wealth of Networks
+                        in 2006.</p>
+                </div>
+            </div>
+        </div> -->
+
+    </section>
+    <script src="js/humburger-menu.js"></script>
+    <script src="js/speakers.js"></script>
+</body>
```
Comment (codepantha) [inline index.html]: - [x] I noticed that your `index.html` page has no footer. Please add a footer to ensure consistency with your `about.html` page 😎

File: css/about.css [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+.navbar {
+  padding: 0 10px;
+}
+
+footer > h5 {
+  text-align: center;
+  font-size: 0.6rem;
+  font-weight: normal;
+}
```
Comment (codepantha) [inline css/about.css:71]: - [x] Your footer should have a dark 🕶️ `background `in desktop view as per the design

File: html/about.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        <div class="evt-d-p">
+            <h2>Please contact us per Email for any further questions about CC Global Summit 2015!</h2>
+            <h5>ccsumit2015@cckorea.org</h5>
+        </div>
+    </section>
+
+    <section class="past-evnt">
+        <div class="container">
+
+            <h2>Creative Commons Global Summit 2015 Logo</h2>
```
Comment (codepantha) [inline html/about.html:73]: - [x] According to the design provided, please add a large photo in this section. Take a look 👇 
image

File: css/about.css [hunk 4]
```diff
@@ -0,0 +1,352 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+:root {
+  --base-color: #272a31;
+  --second-color: #ec5242;
+  --third-color: #d3d3d3;
+  --font: "Poppins", sans-serif;
```
Comment (codepantha) [inline css/about.css]: - [x] Kindly follow the specified font in the design. You should use `Cocogoose, Lato` ✍️

File: index.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+            <h3 class="spk-title">Featured speakers</h3>
+            <div class="line"></div>
+        </div>
+
+        <!-- <div id="ctn-speakers-div" class="ctn-speakers-div">
+            <div id="speaker" class="ctn-spk">
+                <div id="spkr-ctn" class="spk-img-ctn">
+                    <img id="spkr-img" class="img-spk" src="images/spk_1.jpg" alt="speaker image">
+                </div>
+                <div class="ctn-spk-info">
```
Comment (codepantha) [inline index.html]: - [x] Ensure to remove all commented code. Commented codes make your codebase look messy 🙏

File: js/speakers.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+  Object.keys(users).forEach((key) => {
+    const speakerCtn = document.createElement('div');
+    speakerCtn.className = 'ctn-spk';
+
+    const imgCtn = document.createElement('div');
+    imgCtn.className = 'spk-img-ctn';
+
+    const img = document.createElement('img');
+    img.className = 'img-spk';
```
Comment (codepantha) [inline js/speakers.js:58]: - [x] The featured speakers' images do not have an `alt `attribute. The `alt` text is the written copy that appears in place of an image on a webpage if the image fails to load on a user's screen. Please add one for each of your images 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @kosher9 👋, 
# Changes Required ♻️ 

You have made a very great effort in this project. 👏 Though, there are a few changes you need to make in order to complete this milestone, you are very close.

## To highlight 💡 
- Meaningful commit messages ✔️ 
- Good PR ✔️ 
- Good UI design ✔️ 

## Suggested changes
_Check the comments under the review._

- [x] Kindly add a link to the deployed project with Github pages in your README so that everyone can see this amazing project you've built 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: html/about.html (hunk 3 above)
Comment (kosher9) [reply html/about.html:73]: Hi. Please can I add the logo I used for the website there instead of using another picture?

Comment (codepantha) [reply html/about.html:73]: @kosher9 Yes, you may 👍

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @kosher9  👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] Kindly make the footer of this design to match that in the behance design a

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kosher9** [comment]
Hi @Cathella , 
Thank you for your feedback. My previous code reviewer didn't mention that. I thought I can personalize the footer of the website as the one in the preview is related to the topic. My topic is different and as you can see, I have my own logo.

**Comment by prolajumokeoni** [comment]
> Hi @Cathella , Thank you for your feedback. My previous code reviewer didn't mention that. I thought I can personalize the footer of the website as the one in the preview is related to the topic. My topic is different and as you can see, I have my own logo.


Hi @kosher9  the first reviewer requested(https://github.com/kosher9/tech_event/pull/1#discussion_r825298900) you add the footer to the index.html

File: html/about.html [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+        <div class="line"></div>
+
+        <div class="partner">
+            <img class="ptn-img" src="../images/p_1.png" alt="">
+            <img class="ptn-img" src="../images/p_2.png" alt="">
+            <img class="ptn-img" src="../images/p_3.png" alt="">
+            <img class="ptn-img" src="../images/p_4.png" alt="">
+            <img class="ptn-img" src="../images/p_5.png" alt="">
+        </div>
+    </section>
```
Comment (prolajumokeoni) [inline html/about.html]: ## OPTIONAL
- [x] Kindly move this section to the index.html page so it looks like the Behance design👍🏿

File: html/about.html [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <nav class="nav-container">
+            <a class="logo" href="../index.html">Tech<span>Meet</span></a>
+            <img id="menu-button" class="menu-button" src="../images/menu.svg" alt="hamburger menu">
+            <div class="right-ctn">
+                <ul class="nav-menu">
+                    <li><a class="about" href="#">About</a> </li>
```
Comment (prolajumokeoni) [inline html/about.html]: ## OPTIONAL
- kindly change this to index.html so the user can navigate easily to the Home page from the about page

**Comment by kosher9** [comment]
> > Hi @Cathella , Thank you for your feedback. My previous code reviewer didn't mention that. I thought I can personalize the footer of the website as the one in the preview is related to the topic. My topic is different and as you can see, I have my own logo.
> 
> 
> Hi @kosher9  the first reviewer requested](https://github.com/kosher9/tech_event/pull/1#discussion_r825298900) you add the footer to the index.html

Hi @prolajumokeoni . Thank you for your reply. You are right. But it was because I only added a footer to the about page. It was a reminder. But I understand the necessity of following yhe design.

**Review by prolajumokeoni — APPROVED** [review]
Hi @kosher9 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


<br>


all-done-gif


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by prolajumokeoni** [comment]
Please check comments above 👍🏿

**Pull request merged**